src/services/resourceService.ts

```diff
@@ -11,6 +11,11 @@ export class ResourceService extends BaseService {
     this.resourceClient = new ResourceManagementClient(this.credentials, this.subscriptionId);
   }
 
+  public async listDeployments() {
```

Comment:
This is more of a "get" operation and think we should rename to `getDeployments`.  The listing/writing is happening in the plugin.